modules/vaos/app/services/common/jwt_wrapper.rb

```diff
@@ -1,24 +1,40 @@
 # frozen_string_literal: true
 
+require_relative '../../exceptions/vaos/exceptions/configuration_error'
+
 module Common
   class JwtWrapper
     SIGNING_ALGORITHM = 'RS512'
 
+    class ConfigurationError < StandardError; end
+
     attr_reader :expiration, :settings
 
     delegate :key_path, :client_id, :kid, :audience_claim_url, to: :settings
 
-    def initialize(service_settings)
+    def initialize(service_settings, service_config)
       @settings = service_settings
+      @config = service_config
       @expiration = 5
     end
 
     def sign_assertion
       JWT.encode(claims, rsa_key, SIGNING_ALGORITHM, jwt_headers)
+    rescue ConfigurationError => e
+      Rails.logger.error("Service Configuration Error: #{e.message}")
+      raise VAOS::Exceptions::ConfigurationError.new(e, @config.service_name)
     end
 
     def rsa_key
-      @rsa_key ||= OpenSSL::PKey::RSA.new(File.read(key_path))
+      raise ConfigurationError, 'RSA key path is not configured' if key_path.blank?
+
+      raise ConfigurationError, "RSA key file not found at: #{key_path}" unless File.exist?(key_path)
+
+      @rsa_key ||= begin
+        OpenSSL::PKey::RSA.new(File.read(key_path))
```

Comment:
Note: This is where the exception included in the API response (see ticket) was occurring if the RSA key was missing.